nautobot/docs/user-guide/administration/installation/index.md

```diff
@@ -1,24 +1,98 @@
 # Installation
 
-Nautobot can be deployed via Docker (Compose or Kubernetes) or directly onto a supported Linux system:
+Whether you are getting started for the first time, looking to build a local lab or ready to build a production environment, there are a variety of options to find the right fit for your needs with using Nautobot. 
 
-* [Nautobot Docker images](../guides/docker.md) are available via [Docker Hub](https://hub.docker.com/r/networktocode/nautobot) & [GitHub Container Registry](https://github.com/nautobot/nautobot/pkgs/container/nautobot) for use within a containerized environment
-    * Install Nautobot via [Docker Compose](https://github.com/nautobot/nautobot-docker-compose)
-    * [Install Nautobot via Helm Charts](https://docs.nautobot.com/projects/helm-charts/en/stable/) for Kubernetes
-* [Install Nautobot](install_system.md) onto a [RHEL (Red Hat Enterprise Linux)](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) or [Ubuntu](https://ubuntu.com/) virtual machine.
+## Deployment Options
 
-For more information about the Docker tags, Docker configurations, or using container images for your own development environment, see [Nautobot in Docker](../guides/docker.md).
+**Are you checking out Nautobot for the first time and don't want to install anything quite yet?** 
 
-??? info "Supported Platforms"
+You can explore the [Nautobot Always-On Sandbox Environments](https://networktocode.com/nautobot/sandbox-environments/). There is sample data already prepopulated, no installation required, and several Nautobot Apps are pre-installed, with sample data. 
 
-    Nautobot was designed to be a cross-platform application that can run on nearly any system that is able to run the required dependencies. *Only the operating system platforms listed below are officially supported at this time*.
+=== "Bare metal / Virtual Machine"
 
-    - Red Hat flavors of Linux including CentOS 8.2+ and Red Hat Enterprise Linux (RHEL) 8.2+ are supported.
-    - Debian/Ubuntu flavors of Linux including Ubuntu 20.04+ and Debian 11+ are supported.
+    You can [install Nautobot](install_system.md) onto a [RHEL (Red Hat Enterprise Linux)](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) or [Ubuntu](https://ubuntu.com/) virtual machine. 
+    
+    ??? info "Supported Platforms"
 
-    Nautobot *should* work on any POSIX-compliant system including practically any flavor of Linux, BSD, or even macOS, but those are not *officially* supported at this time.
+        Nautobot was designed to be a cross-platform application that can run on nearly any system that is able to run the required dependencies. *Only the operating system platforms listed below are officially supported at this time*.
 
-To begin installing Nautobot, click the link to your preferred deployment approach from the list above.
+        * Red Hat flavors of Linux including CentOS 8.2+ and Red Hat Enterprise Linux (RHEL) 8.2+ are supported.
```

Comment:
If we are doing something here, should we evaluate removing CentOS and put Fedora in here? I'm not seeing much CentOS referenced any more. There are a few other flavors. Not a show stopper, but if we are modifying this, perhaps some look.

Reply:
@jabelk I'm good by RHEL derivatives or something of that nature.